ibm_was/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - IBM WAS
 
+## 1.13.0 / 2022-01-08
+
+* [Fixed] Add comment to autogenerated model files. See [#10945](https://github.com/DataDog/integrations-core/pull/10945).
+* [Fixed] Remove unused config options. See [#10623](https://github.com/DataDog/integrations-core/pull/10623).
```

Comment:
This one is also duplicated

Reply:
The version should be bumped down to 1.12.1

Reply:
I believe that after signing the commit, I can't change the actual code, so I can't change the version to 1.12.1 in `__about__.py`. Would it be ok to keep it at 1.13?

Reply:
No.

Reply:
Oh, add this back https://github.com/DataDog/integrations-core/pull/11076/commits/974333c37ac4eb5807854b57226778fbb68ebf9f#r780627861